### PR TITLE
Add data augmentation to ami recipe

### PR DESCRIPTION
**Pull request by Jzmo**

### CODE REVIEW

File: egs/ami/asr1/run.sh [hunk 1]
```diff
@@ -108,6 +108,13 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
 	PROCESSED_AMI_DIR=${AMI_DIR}
     fi
     local/ami_${base_mic}_data_prep.sh ${PROCESSED_AMI_DIR} ${mic}
+    # data augmentation
+    utils/perturb_data_dir_speed.sh 0.9 data/${mic}/train_orig data/ihm_tmp1
```
Comment (sw005320) [inline egs/ami/asr1/run.sh]: `data/${mic}/train_orig` is first appeared.
I think you need to prepare it.

Comment (Jzmo) [reply egs/ami/asr1/run.sh]: Hi, here `ami_${base_mic}_data_prep.sh` in line 110 automatically create the `data/ihm/train_orig` path when the ${mic} is ihm.

Comment (sw005320) [reply egs/ami/asr1/run.sh]: I see. Thanks.

Comment (Jzmo) [reply egs/ami/asr1/run.sh]: Also, do you think I need to upload the model files?

Comment (sw005320) [reply egs/ami/asr1/run.sh]: This is very helpful. I sent an invitation to Google Drive. Please upload your model and modify the link accordingly.

**Comment by sw005320** [comment]
Many thanks!
Can you also try specaugment?
If you check https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh#L23, you could see how you can use it.
It would be great if you check the following setups
speed purturb. vs. specaug. vs. speed purturb+specaug

**Pull request merged**